server/.eslintignore

```diff
@@ -0,0 +1 @@
+migrations
```

Comment:
@nikitasakov you have to config your text editor to leave empty line at the end of each file. Let me knot if it's unclear

Reply:
Such a problem is in several files

Reply:
`"ensure_newline_at_eof_on_save": true,`

for sublime ^